src/components/Upload/index.js

```diff
@@ -17,7 +17,7 @@ const handleUpload = async (content) => {
   }
   const session = await getSession();
   return fetchJson(
-    `${process.env.TWOOPSTRACKER_API_URL}/lists/upload`,
+    `${process.env.NEXT_PUBLIC_TWOOPSTRACKER_API_URL}/lists/upload`,
```

Comment:
Been a while since I've looked at this but I think the design was:

We don't want to expose the API backend directly to the front-end. Front-end should post to `/api/lists` so that authentication is done via NextAuth, etc., and then `/api/list/` will communicate with  `${process.env.TWOOPSTRACKER_API_URL}/lists/upload` return result to FE.